app/src/main/java/de/danoeh/antennapod/activity/MainActivity.java

```diff
@@ -379,7 +465,8 @@ protected void onStop() {
     public boolean onOptionsItemSelected(MenuItem item) {
         if (drawerToggle.onOptionsItemSelected(item)) {
             return true;
-        } else if (item.getItemId() == android.R.id.home) {
+        } else
+        if (item.getItemId() == android.R.id.home) {
```

Comment:
it's very strange for the 'else' and 'if' to be split across lines. It could give someone the wrong idea.